Makefile-os

```diff
@@ -11,21 +16,44 @@ help_submake:
 
 .PHONY: update_docker
 update_docker: ## update all the docker images
-	docker-compose exec worker make update_deps
-	docker-compose exec web make update
+	docker-compose exec --user olympia worker make update_deps
+	docker-compose exec --user olympia web make update
 	docker-compose restart web
 	docker-compose restart worker
 
 .PHONY: shell
 shell: ## connect to a running addons-server docker shell
-	docker-compose exec web bash
+	docker-compose exec --user olympia web bash
 
-.PHONY: shell
+.PHONY: rootshell
 rootshell: ## connect to a running addons-server docker shell with root user
 	docker-compose exec --user root web bash
 
+.PHONY: create_env_file
+create_env_file:
+	echo "UID=${UID}\nGID=${GID}" > .env
+
+.PHONY: initialize
+initialize: create_env_file
```

Comment:
I'm not super happy about us overriding the name of `initialize` recipe, as it'll mean it does different things depending on whether you're inside or outside of docker.  For updates we have `update_docker` (in Makefile-os) and `update` (in Makefile-docker) - maybe this should be called `initialize_docker`?  (alternatively, have `update` and `initialize` here, and the container specific ones be called `update_container` and `initializer_container` or similar?)